charts/vela-core/templates/defwithtemplate/gateway.yaml

```diff
@@ -125,6 +133,12 @@ spec:
 
         	// +usage=Specify a pathType for the ingress rules, defaults to "ImplementationSpecific"
         	pathType: *"ImplementationSpecific" | "Prefix" | "Exact"
+
+        	// +usage=Specify the annotations to be added to the gateway
```

Comment:
I think "ingress" is better than "gateway" in the comment.

Reply:
I will modify it